playground/mocks/data/idp/idx/authenticator-verification-okta-verify-factor-list.json

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
`autochallenge` is not required for this view. So this can be deleted and found an existing json file (`authenticator-verification-data-ov-only-without-device-known`) that can be used.